simple/schema.py

```diff
@@ -32,6 +33,25 @@ class Instruments(Base):
 # -------------------------------------------------------------------------------------------------------------------
 # Hard-coded enumerations
 
+class Regime(enum.Enum):
+    """Enumeration for spectral type regime"""
+    gammaray = 'gammaray'
+    xray = 'xray'
+    ultraviolet = 'ultraviolet'
+    optical = 'optical'
+    infrared = 'infrared'
+    millimeter = 'millimeter'
+    radio = 'radio'
+
+
+class Gravity(enum.Enum):
+    """Enumeration for gravity"""
+    a = 'alpha'
+    b = 'beta'
+    g = 'gamma'
+    bg = 'beta/gamma'
+    unknown = 'unknown'
```

Comment:
What about the ones from NIR spectra? e.g. vl-g, int-g, fld-g.
There's also delta for optical

Reply:
Yep, good points- I don't know all the possibilities so it's good to get feedback like this!